### DISCUSSION TITLE
What is the numbering ?

### DISCUSSION THREAD

**Discussion by Jeffr-K** [ohmyzsh/ohmyzsh#10893] [Help]
I installed agnoster theme. but I think the number is not pretty on my eyes and I don't know what is the >R> meaning.

how do I fix them?

<img width="347" alt="Screen Shot 2022-04-27 at 2 41 34 PM" src="https://user-images.githubusercontent.com/89901793/165449345-11975a58-efb8-4fa4-b5da-b3fec6e5060c.png">

**Comment by mcornella** [comment]
Those are commit ids, also called commit hashes. It means you have checked out a specific commit and not a branch. There's nothing to be fixed, once you go back to a branch that'll be displayed instead.

The `>R>` means you're in the middle of a `git rebase`.
